web/annotation_editor_layer_builder.css

```
@@ -94,21 +99,17 @@
   outline: none;
 }
 
-.annotationEditorLayer .freeTextEditor:focus-within {
-  outline: var(--focus-outline);
-}
-
-.annotationEditorLayer .inkEditor:not(:focus) {
+.annotationEditorLayer .inkEditor.disabled:not(.selectedEditor) {
```

Is the *last* part actually necessary here, or does the following work (given the rule just below which override it when necessary)?
```suggestion
.annotationEditorLayer .inkEditor.disabled {
```